packages/utils/other-utils/src/interpolate-colors.js

```diff
@@ -95,10 +95,6 @@ export function getCellColors(params) {
     theme,
   } = params;
   if (cellSetSelection && cellSets) {
-    // Cell sets can potentially lack set colors since the color property
-    // is not a required part of the schema.
-    // The `initializeSets` function fills in any empty colors
-    // with defaults and returns the processed tree object.
     return treeToCellColorsBySetNames(cellSets, cellSetSelection, cellSetColor, theme);
```

Comment:
this comment looked irrelevant, so I decided to delete it. Let me know if this is not the case though -- I will bring it back.